stylesheets/index.css

```diff
@@ -1,38 +1,327 @@
 @import url("/stylesheets/style.css");
-@import url('https://fonts.googleapis.com/css2?family=Festive&family=Heebo:wght@200;300;400;500;600;800;900&family=Rubik:wght@300;400;600;700&display=swap');
 
-.gh-navbar{
-    box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
-    background-color: #f8f9fa;
-    color: #212529;
-    display: flex;
+body {
+  background-color: var(--gh-bg1);
 }
 
-.gh-nav-head{
-font-family: 'Festive', cursive;
-font-size: 2.5rem;
-margin: 0.5rem;
+.gh-navbar {
+  background-color: var(--gh-bg2);
+  color: var(--gh-blk);
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  top: 0rem;
+  z-index: 99;
 }
 
-.gh-search-holder{
-    border: 1px solid  #9D9EAD;
-    background-color: #f8f9fa;
-    margin: auto;
+.gh-nav-head {
+  font-family: "Poppins", sans-serif;
+  font-size: 1.8rem;
+  margin: 0.3rem 0.6rem;
+  display: flex;
+  align-items: center;
+  gap: 0.2rem;
+  color: var(--gh-blk);
 }
 
-.gh-search{
-    background-color: #f8f9fa;
+.gh-search-holder {
+  border: 1px solid var(--gh-gray);
+  background-color: var(--gh-bg2);
+  border-radius: 0.2rem;
 }
 
-::-ms-input-placeholder { 
-    font-size: 0.8rem; 
-    font-weight: 900;
-    margin: 2rem ;   /* Microsoft Edge */
+.gh-nav-icon-wrap {
+  margin: 0.5rem;
+  display: flex;
 }
-::placeholder { 
-    font-size: 0.8rem; 
-    margin: 2rem;    /* Firefox, Chrome, Opera */
+
+.gh-search {
+  background-color: var(--gh-bg2);
+}
+
+.gh-nav-icon-chip {
+  margin: 0 0.4rem;
+  padding: 0.6rem;
+}
+
+.gh-nav-icon {
+  font-size: 1.3rem;
+  color: var(--gh-blk);
+}
+
+.gh-nav-badge {
+  background-color: var(--gh-blk);
+}
+
+.fa-seedling {
+  color: var(--gh-blk);
+}
+
+::-ms-input-placeholder {
+  font-size: 0.8rem;
+  font-weight: 900;
+  margin: 2rem; /* Microsoft Edge */
+}
+::placeholder {
+  font-size: 0.8rem;
+  margin: 2rem; /* Firefox, Chrome, Opera */
+}
+
+.wrapper {
+  background-color: var(--gh-bg2);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  box-shadow: var(--gh-footer-shadow);
+  gap: 1rem;
+  padding-top: 6rem;
+}
+
+.tab-section {
+  border-bottom: 2px solid var(--gh-bg2);
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
These lines of code can be used frequently so you can make a separate utility class for it and then pass it into your HTML file.

Reply:
Will implement it